Index: libnl-3.2.25/lib/route/route_obj.c
===================================================================
--- libnl-3.2.25.orig/lib/route/route_obj.c
+++ libnl-3.2.25/lib/route/route_obj.c
@@ -1308,7 +1308,7 @@
 	.oo_attrs2str		= route_attrs2str,
 	.oo_id_attrs		= (ROUTE_ATTR_FAMILY | ROUTE_ATTR_TOS |
 				   ROUTE_ATTR_TABLE | ROUTE_ATTR_DST |
-				   ROUTE_ATTR_PRIO),
+				   ROUTE_ATTR_PRIO | ROUTE_ATTR_MULTIPATH),
 };
 /** @endcond */
 
